cmd/account/governance.go

```diff
@@ -0,0 +1,193 @@
+package account
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+
+	governance "github.com/oasisprotocol/oasis-core/go/governance/api"
+	upgrade "github.com/oasisprotocol/oasis-core/go/upgrade/api"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	"github.com/oasisprotocol/cli/cmd/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+var (
+	govCmd = &cobra.Command{
+		Use:     "governance",
+		Short:   "Governance operations",
+		Aliases: []string{"gov"},
+	}
+
+	govSubmitProposalCmd = &cobra.Command{
+		Use:   "submit-proposal",
+		Short: "Submit a governance proposal",
```

Comment:
Since we're using term "submit" in the transaction command for broadcasting existing transaction, let's mention explicitly this is a *new* proposal.

```suggestion
		Short: "Submit new governance proposal",
```

Reply:
This is because the transaction method is called "submit proposal". I changed the commands now to "create proposal".